php/php.phpunit/src/org/netbeans/modules/php/phpunit/commands/PhpUnit.java

```diff
@@ -406,7 +407,12 @@ private TestParams getTestParams(PhpModule phpModule, TestRunInfo runInfo) throw
                 // standard suite
                 // #218607 - hotfix
                 //params.add(SUITE_NAME)
-                params.add(getNbSuite().getAbsolutePath());
+                if (PhpUnitPreferences.getPhpUnitVersion(phpModule) == PhpUnitVersion.PHP_UNIT_9) {
+                    params.add(getNbSuite().getAbsolutePath());
+                } else {
+                    // GH-5790 we can use NetBeansSuite.php no longer with PHPUnit 10
+                    params.add(FileUtil.toFile(startFiles.get(0)).getAbsolutePath());
```

Comment:
And just to verify - is this the final change? It does not seem to me to be; or am I wrong? Thanks.

Reply:
I am asking because it is - at least, I believe it is - possible to select more test files (using CTRL) and run them (so, test them using PHPUnit). I hope I am not wrong...

Reply:
Yes, you are right. However, I'm not sure the way to run multiple files...
We could do it via `NetBeansSuite.php`, so far, right?

```
php vendor/bin/phpunit --help
PHPUnit 10.1.1 by Sebastian Bergmann and contributors.

Usage:
  phpunit [options] UnitTest.php
  phpunit [options] <directory>
```

Reply:
Exactly. This is the reason why we had to use `NetBeansSuite.php`. I am afraid we need to investigate how/whether this is supported in PHPUnit now...